docs/sources/features/datasources/influxdb.md

```diff
@@ -64,16 +64,16 @@ Identifier | Description
 
 {{< docs-imagebox img="/img/docs/v45/influxdb_query_still.png" class="docs-image--no-shadow" animated-gif="/img/docs/v45/influxdb_query.gif" >}}
 
-You find the InfluxDB editor in the metrics tab in Graph or Singlestat panel's edit mode. You enter edit mode by clicking the
-panel title, then edit. The editor allows you to select metrics and tags.
+You can find the InfluxDB editor in the metrics tab in a Graph or Singlestat panel's edit mode. 
+You can enter edit mode by clicking the panel title, then edit. The editor allows you to select metrics and tags.
 
 ### Filter data (WHERE)
 To add a tag filter click the plus icon to the right of the `WHERE` condition. You can remove tag filters by clicking on
 the tag key and select `--remove tag filter--`.
 
 **Regex matching**
 
-You can type in regex patterns for metric names or tag filter values, be sure to wrap the regex pattern in forward slashes (`/`). Grafana
+You can type in regex patterns for metric names or tag filter values. Be sure to wrap the regex pattern in forward slashes (`/`). Grafana
 will automatically adjust the filter tag condition to use the InfluxDB regex match condition operator (`=~`).
```

Comment:
```suggestion
automatically adjusts the filter tag condition to use the InfluxDB regex match condition operator (`=~`).
```

Reply:
I committed this suggestion separately because the github UI thinks it's outdated. Looks good to me though!